### ISSUE TITLE
Users Can Export Inventory Data to CSV

### ISSUE THREAD

**Issue by candyj0311** [MobilityLabs/pdredesign-server#114]
Once an inventory is completed, facilitators and participants can download the inventory data to a .csv file. We already have this functionality for the Readiness Assessment.

**Comment by candyj0311** [comment]
Depends on #184.

**Comment by MakotoTheKnight** [comment]
Arrays can be brought back as a list.  Any arrays brought back that aren't strings (i.e. associated products) will need to have their name brought back instead.

**Comment by MakotoTheKnight** [comment]
We can get the backend of this started now without having the frontend piece defined just yet.

**Comment by juliosantos** [comment]
This is a backend piece, we need a new task to capture the frontend.

**Comment by juliosantos** [comment]
I initially understood this as "export inventories to CSV". "Inventory Data" refers to what, Data Entries?

**Comment by juliosantos** [comment]
@candyj0311 ^

**Comment by candyj0311** [comment]
Data and product entries.

**Comment by juliosantos** [comment]
OK. Presumably those would be two separate CSVs, since data and product entries don't share the same fields (headers in the CSV file). Right?

**Comment by juliosantos** [comment]
@candyj0311 says:
> So let's go with 2 files and the zipped version can be an enhancement since we're not long on time. I'll talk to Cat about adjusting the mockups to make this clear.

**Comment by juliosantos** [comment]
@candyj0311 I will create two endpoints then, one per file to get, so something like `inventories/1/product_entries.csv` and `inventories/1/data_entries.csv`.

**Comment by juliosantos** [comment]
@MakotoTheKnight when you say "list", what do you mean?

**Comment by HeyCat** [comment]
Mock added to #184.

**Comment by MakotoTheKnight** [comment]
@juliosantos:  The idea is, for responses that are tied to array columns, we'll need to bring those back as a list of strings.  It will likely take the form of its own nested CSV column (i.e. for product types, you would have "Curriculum Products","Human Resources" as part of the CSV data.).
